pkg/repository/udmrepo/kopialib/lib_repo_test.go

```diff
@@ -264,177 +263,6 @@ func TestMaintain(t *testing.T) {
 	}
 }
 
-func TestWriteInitParameters(t *testing.T) {
```

Comment:
Are we covering all the tests in this PR that are being deleted here ?

Reply:
These are just moved over to pkg/repository/udmrepo/kopialib/repo_init.go and its _test.go